src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -142,7 +141,7 @@
         </service>
 
         <service id="twig.controller.preview_error" class="Symfony\Bundle\TwigBundle\Controller\PreviewErrorController" public="true">
-            <argument type="service" id="http_kernel" />
+            <argument />
```

Comment:
that is still a broken service definition here. The compiler pass should not handle adding the argument, but should handle removing the service instead (if we don't have HttpKernel, we don't really need the error preview controller)